### ISSUE TITLE
Meetup in Barcelona

### ISSUE THREAD

**Issue by Ferdinandreyes** [w3f/Web3-collaboration#36]
I am Ferran, from Caelum Labs, a Barcelona-based company designing and developing Blockchain solutions.
In parallel with that, we dynamize the blockchain community organizing [Devcentralised meetup](https://www.meetup.com/es/Devcentralised-Developing-DApps-Barcelona/), with almost 600 members at this moment.

We would like to host one Polkadot Meetup in Barcelona, focused on the technology and for devs.

**How many people:** We count on one person to organize the meetup but also a team that can help with it.

**Venue:** We have space in our office to organize the meetup but we can find other places if necessary.

**Do you need help from W3F:** We would need a team member to do a presentation of the technology behind Polkadot and able to solve tech doubts from the community. We can organize it also as a workshop or focused in some of the polkadot network features.

**Do you need help from the community:** Would be nice to have promotion from Polkadot community.

**Comment by EdwardAThomson** [comment]
Hey Ferran (@Ferdinandreyes) Great to hear from you! This sounds like a very strong meetup group and would be great to present Polkadot to them. We actually just heard from some in Barcelona community and I'd like to point you towards the current open issue for Barcelona: 

https://github.com/w3f/Web3-collaboration/issues/17

It is easier if we can all collaborate in that one! Thank you.

I would also suggest that you fill in our coordination form as it helps us to gather email addresses privately (etc):

[Collaboration form](https://docs.google.com/forms/d/e/1FAIpQLSe-hbgzMHkwYLCy9-0NLmnpIJdD-nwq_vqNwba9JiiKFmZmZg/viewform)
